internal/storage/bucket_handle.go

```diff
@@ -321,11 +322,31 @@ func (b *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsRequ
 		// https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/vendor/cloud.google.com/go/storage/storage.go#L1304
 		// https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/vendor/cloud.google.com/go/storage/http_client.go#L370
 		if attrs.Prefix != "" {
-			list.CollapsedRuns = append(list.CollapsedRuns, attrs.Prefix)
+			if util.IsUnsupportedObjectName(attrs.Prefix) {
+				// Do not list unsupported directories such as those including
```

Comment:
fix this comment here. Make it shorter e.g. 

```
Ignore GCS directories containing `//` in their names as they have a special meaning in linux file-system and can not be represented by the GCSFuse. 
As an example, GCS can have both the objects a//b and a/b at the same time as two different objects, but to linux FS, both paths are the same. So, GCSFuse will ignore object a//b to avoid causing `input/output error` in linux filesystem.
```